swap/swap.go

```diff
@@ -256,13 +256,46 @@ func (s *Swap) createCheque(peer enode.ID) (*Cheque, error) {
 }
 
 //GetPeerBalance returns the balance for a given peer
-func (swap *Swap) GetPeerBalance(peer enode.ID) (int64, error) {
-	swap.lock.RLock()
-	defer swap.lock.RUnlock()
-	if p, ok := swap.balances[peer]; ok {
-		return p, nil
+func (s *Swap) GetPeerBalance(peer enode.ID) (int64, error) {
```

Comment:
Just as a suggestion, to remove Get prefixes on methods. This is somehow a general practice, but not required if it makes sense to have it. In this cases, I think we can remove it.

Reply:
Is this somewhere in the `go` guidelines? I agree in this case we can remove it (it was there before, added by me, sorry @mortelli )

Reply:
Done! Thanks.

Reply:
> Is this somewhere in the `go` guidelines? I agree in this case we can remove it (it was there before, added by me, sorry @mortelli )

Thanks, I've read it here https://golang.org/doc/effective_go.html#Getters. Seams nice, but not crucial.